easy-overlay.js

```diff
@@ -65,7 +65,7 @@ var easyOverlay=(function(){
 		}
 		$overlay.remove();
 		if (count>1){
-			$('#overlay'+(count-1)).css({overflow: overflows[count]});
+			$('#overlay'+(count-1)).css(overflows[count] ? overflows[count] : {});

```

Comment:
If this is optional better would be to make the whole line conditional on the presence of the overflows key - otherwise `.css({})` does nothing but looks like it might do.